Dockerfile

```
@@ -41,36 +41,33 @@ RUN apt-get clean \
   && apt-get update \
   && apt-get install -y \
   build-essential python3.9 python3-pip git cmake make ruby rake autoconf automake \
```

Why do you install `make`?  It is not required AFAIK? My `Dockerfile` does not install it and everything works as expected?

And could we prettify this and sort the commands?

```suggestion
  && apt-get install -y \
    autoconf \
    automake \
    build-essential \
    cmake \
    git \
    python3 \
    python3-pip \
    rake \
    ruby \
```